omnibus/files/private-chef-cookbooks/private-chef/recipes/bootstrap.rb

```diff
@@ -1,70 +1,59 @@
 #
-# Author:: Adam Jacob (<adam@opscode.com>)
-# Copyright:: Copyright (c) 2011 Opscode, Inc.
+# Author:: Adam Jacob <adam@chef.io>
+# Copyright:: Copyright (c) 2011-2015 Chef Software, Inc.
 #
 # All Rights Reserved
 #
 
-require 'securerandom'
-
 # Enterprise Addon Install
-# On a new install, download and install all of the Chef
-# addons from package repositories. Only perform this
-# action during a fresh install, and not during an upgrade.
+include_recipe "private-chef::opscode-erchef"
```

Comment:
Why do we have to require the `opscode-erchef` recipe here? IIRC, this is now the only place where we include recipes for individial services. There's a loop in the base recipe the includes all the services that are required. My concern here is that we'll start up and configure erchef even if the config has explicitly turned it off.